cloudflare/resource_cloudflare_spectrum_application_test.go

```diff
@@ -225,63 +225,69 @@ func testAccManuallyDeleteSpectrumApplication(name string, spectrumApp *cloudfla
 func testAccCheckCloudflareSpectrumApplicationConfigBasic(zoneName, ID string) string {
 	return fmt.Sprintf(`
 resource "cloudflare_spectrum_application" "%[2]s" {
-  zone_id  = "${data.cloudflare_zone.test.id}"
+  zone_id  = "${lookup(data.cloudflare_zones.test.zones[0], "id")}"
```

Comment:
this isn't the prettiest thing out there but it works. open to suggestions on improvements

Reply:
`data.cloudflare_zones.test.zones.*.id[0]` ?